rdmd.d

```diff
@@ -713,9 +725,178 @@ private string[string] getDependencies(string rootModule, string workDir,
         exit(depsExitCode);
     }
 
+    if (!hasDependencies)
+        std.file.remove(rootModule);
+
     return dryRun ? null : readDepsFile();
 }
 
+bool needsInspection(string file)
+{
+    import std.utf : byChar;
+
+    foreach (match; file.matchAll(ctRegex!(`import[^a-zA-Z]*([^;]*);`, "s")))
```

Comment:
Don't we want `import` to not be preceded by an alphanumeric and always followed by at least one whitespace?